building/trusted_applications.rst

```diff
@@ -299,3 +305,66 @@ sequence:
             return TEE_ERROR_NOT_SUPPORTED;
         }
     }
+
+.. _TASign:
+
+Signing of TAs
+**************
+
+All :ref:`REE Filesystem Trusted Applications<ree_fs_ta>` need to be signed. The signature is verified by :ref:`optee_os` upon loading of the TA. Within the :ref:`optee_os` source is a directory ``keys``. The public part of ``keys/default_ta.pem`` will be compiled into the :ref:`optee_os` binary and the signature of each TA will be verified against this key upon loading.
```

Comment:
This and everything below needs to be wrapped on 80 character (like it's done in all other rst files in this git).

Reply:
Done.